ee/clickhouse/sql/cohort.py

```diff
@@ -49,3 +49,33 @@
     AND person.is_deleted = 0
     AND id IN ({cohort_filter})
 """
+
+GET_DISTINCT_ID_BY_ENTITY_SQL = """
+SELECT distinct_id FROM events WHERE team_id = %(team_id)s {date_query} AND {entity_query}
+"""
+
+GET_PERSON_ID_BY_ENTITY_COUNT_SQL = """
+SELECT person_id FROM events 
+INNER JOIN (
+    SELECT person_id,
+        distinct_id
+    FROM (
+            SELECT *
+            FROM person_distinct_id
+            JOIN (
+                    SELECT distinct_id,
```

Comment:
We already have SQL to do this I believe.

Reply:
yup good point